persistence/src/main/java/edu/unc/lib/dl/persist/services/edit/EditTitleService.java

```diff
@@ -153,47 +170,46 @@ public void setOperationsMessageSender(OperationsMessageSender operationsMessage
     /**
      *
      * @param mods the mods record to be edited
-     * @return current title value as a string
+     * @return true if mods has title
      */
-    private String getOldTitle(Element mods) {
-        String oldTitle;
+    private Boolean hasExistingTitle(Element mods) {
+        Boolean hasOldTitle;
```

Comment:
you can skip going through a variable and immediately return true or false (its fine to return in the `catch` block). Also, the return type probably doesn't need to be a `Boolean` in this case, the primative `boolean` should be fine.